pkg/core/consensus/user/sortition.go

```diff
@@ -115,8 +115,8 @@ func (p Provisioners) CreateVotingCommittee(seed []byte, round uint64, step uint
 				break
 			}
 
-			if m.Stakes[i].StartHeight > round || m.Stakes[i].EndHeight < round {
-				subtractFromTotalWeight(W, m.Stakes[i].Amount)
+			if m.Stakes[i].CreatedAt > round || m.Stakes[i].Eligibility < round {
```

Comment:
@autholykos I'm concerned about this, not sure if it's correct anymore with respect to what's now understood as ["valid"](https://github.com/dusk-network/rusk/blob/master/contracts/stake/src/stake.rs#L59-L61)

Would appreciate any insight on the matter...

cc/ @ureeves @goshawk-3

Reply:
Since expiration is now removed, the logic around it should be removed here as well.

`eligibility` is the block at which a stake starts being valid. Assuming that `round` is the current block (unsure about this), the logic should then be exceedingly simple:

```go
if round >= m.Stakes[i].Eligibility {
    // stake is valid here
}
```